qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -41,6 +43,22 @@
 logger = logging.getLogger(__name__)
 
 
+@bind_schema(BackendJobsLimitResponseSchema)
+class BackendJobsLimit(BaseModel):
```

Comment:
Since `IBMQBackend.jobs_limit()` is a user facing method, I thought it best to create a model so users could access the attributes directly, instead of receiving a dictionary with the fields. However, I'm not sure on the best place (*file*) to place this model in?